app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionTabs/makeDecisionTabs.jsx

```diff
@@ -0,0 +1,179 @@
+  const [animationSuggest, setAimationSuggest] = useState(true);
+  const suggestVariant = ['one', 'two', 'three'];
+  const [selectDefectTab, machineLeariningTab, copyFromHistoryLineTab] = tabs;
+  const source = modalState.suggestChoice;
+
+  const selectMachineLearningSuggestionItem = (index, itemId) => {
+    if (itemId && itemId === source.id && machineLeariningTab.isOpen) {
+      return;
+    }
+    toggleTab(MACHINE_LEARNING_SUGGESTIONS);
+    setSelectedMLSuggest(index);
+
+    if (index === 0) {
+      setAimationSuggest(false);
+    }
+
+    const { testItemResource, logs, suggestRs } = suggestedItems.find(
+      (item) => item.testItemResource.id === itemId,
+    );
+    setModalState({
+      ...modalState,
+      decisionType: MACHINE_LEARNING_SUGGESTIONS,
+      issueActionType: '',
+      suggestChoice: { ...testItemResource, logs, suggestRs },
+    });
+  };
+
+  const selectSideItem = (tab) => {
+    toggleTab(tab);
+
+    setModalState({
+      ...modalState,
+      decisionType: tab,
+    });
+  };
+  return (
+    <div className={cx('make-decision-tabs')}>
+      {!isBulkOperation && (
+        <div className={cx('tabs')}>
+          <div
+            onClick={() => selectSideItem(SELECT_DEFECT_MANUALLY)}
+            className={cx('side-block', {
+              'side-block-active': selectDefectTab.isOpen,
+            })}
+          >
+            <p className={cx('side-block-title')}>{formatMessage(messages.manual)}</p>
+            <p className={cx('side-block-text')}>{formatMessage(messages.selection)}</p>
+          </div>
+          <div className={cx('central-block')}>
+            {loadingMLSuggest && (
+              <div className={cx('central-block-default')}>
+                <div className={cx('preloader')}>
+                  <DottedPreloader />
+                </div>
+                <p className={cx('suggest-text')}>{formatMessage(messages.analyzingSuggestions)}</p>
+              </div>
+            )}
+            {(suggestedItems.length === 0 || !isAnalyzerAvailable) && !loadingMLSuggest && (
+              <div className={cx('central-block-default')}>
+                <p className={cx('no-suggest-text')}>
+                  {formatMessage(
+                    isAnalyzerAvailable ? messages.noSuggestions : messages.noAnalyzer,
+                  )}
+                </p>
+              </div>
+            )}
+
+            {isAnalyzerAvailable &&
+              suggestedItems.map(({ suggestRs, testItemResource }, index) => (
+                <div
+                  key={testItemResource.id}
+                  onClick={() => selectMachineLearningSuggestionItem(index, testItemResource.id)}
+                  className={cx(`${suggestVariant[suggestedItems.length - 1]}-suggest-item`, {
+                    [`${suggestVariant[suggestedItems.length - 1]}-suggest-item-active`]:
+                      machineLeariningTab.isOpen && selectedMLSuggest === index,
```

Comment:
These items hsould fill the space between manuadl and history tabs. So why you are specifiing separate states for them instead of using relative width calculation?